lib/straight/transaction.rb

```diff
@@ -0,0 +1,14 @@
+module Straight
+  Transaction = Struct.new(:tid, :amount, :confirmations, :block_height) do
```

Comment:
Why you don't like class description as 
```
class Transaction < Struct.new(:tid, :amount, :confirmations, :block_height)
  class << self
  end
end
```
I think we should do it in the project everywhere equally.